docs/content/v0.6/2.configuration/2.nuxt-config.md

```diff
@@ -132,7 +132,7 @@ type ProviderLocal = {
    */
   endpoints?: {
     /**
-     * What method and path to call to perform the sign-in. This endpoint must return a token that can be used to authenticate subsequent requests.
+     * What method and path to call to perform the sign-in. This endpoint must return a token that can be used to authenticate subsequent requests. This page will also not be blocked by the global middleware.
```

Comment:
this is the `endpoints:` specification

Reply:
I don't think so, it talks about signin and the `endpoints` have a description above it.